op-deployer/justfile

```diff
@@ -87,16 +87,16 @@ _validate_rpc:
 pre-deploy: build build-contracts
 
 # Run with:
-# PRIVATE_KEY=0x1234.... NETWORK=sepolia just deploy-opcm 'op-contracts/v4.0.0-rc.8'
-deploy-opcm release="dev" locator="$DEFAULT_LOCATOR": _validate_rpc
+# PRIVATE_KEY=0x1234.... NETWORK=sepolia just deploy-opcm
+deploy-opcm locator="$DEFAULT_LOCATOR": _validate_rpc
   #!/bin/bash
   echo "Using artifacts locator: {{locator}}"
   go run ./cmd/op-deployer bootstrap implementations \
     --l1-rpc-url $ETH_RPC_URL \
     --private-key $PRIVATE_KEY \
     --protocol-versions-proxy $PROTOCOL_VERSIONS_PROXY \
     --superchain-config-proxy $SUPERCHAIN_CONFIG_PROXY \
-    --l1-proxy-admin-owner $L1_PROXY_ADMIN_OWNER \
+    --upgrade-controller $L1_PROXY_ADMIN_OWNER \
```

Comment:
Interesting that there is no `--l1-contracts-release {{release}} ` to remove. I guess this was tidied up on `develop` after the `proposal/v5.0.0` branch was cut?